custom-recipes/compute-results-statistics/recipe.json

```diff
@@ -0,0 +1,53 @@
+{
+    "meta": {
+        "label": "AB testing statistics",
+        "description": "Compute statistics to analyze the result of an AB test",
+        "icon": " icon-subscript"
+    },
+    "selectableFromDataset": "results",
+    "kind": "PYTHON",
+    "inputRoles": [
+        {
+            "name": "results",
+            "label": "Experiment results",
+            "description": "This dataset should contain  the experiment's results at a user level. There should be group column and a conversion column.",
```

Comment:
Double space between `contain` and `the`

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/2/commits/c9888533223d12828b49e2a8ca56a4030fb93a6d